_projects/ft_workflow_project.md

```diff
@@ -102,26 +102,42 @@ derive a clear model for replication, as there are many ways to implement it,
+
+We provide a detailed analytical study for all of these scenarios, with formulas to decide, for each scenario, the optimal parameters as a function of the error rate, checkpoint cost, and platform size. We also report a set of extensive simulation results that nicely corroborates the analytical model.
+
+While the previous work had focused on applications for which we can decide at which frequency we can checkpoint, and our aim has been to find the optimal checkpointing period, we have also initiated the study for linear chains of parallel tasks. The aim is then to decide which tasks to checkpoint and/or replicate. In this case, we have
+provided an optimal dynamic programming algorithm, and an extensive set of simulations to assess (i) in which scenarios checkpointing performs better than replication, or vice-versa; and (ii) in which scenarios the combination of both methods is useful, and to what extent.
 
-Work has been focused toward using replication as a detection and correction mechanism for Silent Data Corruptions (SDC).
-Although other detection techniques exist for HPC applications, based on algorithms (ABFT), invariant preservation or data analytics, replication remains the most transparent and least intrusive technique.
-In this work, replication is combined with checkpoiting to enable rollback and recovery when forward recovery is not possible, which occurs when too many replicas are corrupted.
-The goal is to find the right level (duplication, triplication or more) of replication needed to efficiently detect and correct silent errors at scale.
-As of today, we provide a detailed analytical study with formulas to decide the optimal parameters as a function of the error rate, checkpoint cost, and platform size.
 
 
 ## Visits and meetings
 
-{% person cavelan_a %} visited {% person cappello_f %} in Chicago for three 
-months (march-april-may 2016) to initiate the project. We are working 
-in close collaboration to make progress.
+{% person cavelan_a %} visited {% person cappello_f %} in Chicago for three months (March, April, and May 2016) to initiate the project. Furthermore, we have been meeting regularly in the previous years. In particular,
+we have been attending the SC conference (November 2016
+and November 2017), where we had extensive discussions to make progress. 
+We represented the JLESC at the Inria booth during these conferences. 
+
+While not meeting in person, we have stayed in close collaboration through regular Skype meetings,
+which allowed us to make progress on the project. 
+
 
 ## Impact and publications
 
-Two papers have been accepted to FTXS'17 {% cite benoitEtAl2017identifying --file external/ft_workflow_project.bib
-%},{% cite benoitEtAl2017optimal --file external/ft_workflow_project.bib
-%}.
+Two papers have been accepted to FTXS'17 {% cite benoitEtAl2017identifying --file external/ft_workflow_project.bib %},{% cite benoitEtAl2017optimal --file external/ft_workflow_project.bib %}.
```

Comment:
Are those publications coming from this JLESC project, with partners from JLESC labs? If so, please add it to the [jlesc.bib](https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib) and cite from there.

Reply:
Yes they are. I'll try to do the modification.